docs/fwupd.conf.md

```diff
@@ -216,6 +216,10 @@ The `[fwupd]` section can contain the following parameters:
   Create virtual test devices and remote for validating daemon flows.
   This is only intended for CI testing and development purposes.
 
+**FixRemotes={{FixRemotes}}**
```

Comment:
Conceptually I agree with this; but isn't it a bit presumptuous to enable by default?

I'm wondering if we need some way to test the remote actually supports xz before doing such a migration and push it up to the client to indicate it can be done over dbus.

Reply:
Maybe we allow-list the auto-fix behaviour to remotes that contain `fwupd.org`? Maybe if we do that then we don't need the config option at all?

Reply:
> that contain fwupd.org

maybe also "have an ID starting with `lvfs`" to cope with mirrors too

Reply:
Yeah I think that's a safer approach.

Reply:
Fixed.